.travis.yml

```diff
@@ -49,6 +49,7 @@ script:
 - cd Build
 - cmake .. -DBUILD_LAGOM=1 -DALL_THE_DEBUG_MACROS=1
 - make -j2
+- Meta/check-symbols.sh
```

Comment:
@itamar8910 @awesomekling Because of the `cd Build` a few lines earlier, Travis is in the `Build` folder at this point, so naturally it tries to search for the script at `Build/Meta/check-symbols.sh`. That's why all builds are currently failing. :P